libiocage/lib/Jail.py

```diff
@@ -60,19 +62,24 @@ class Jail:
             release (stored in `zpool/iocage/base/<RELEASE>)
     """
 
-    def __init__(self, data={}, zfs=None, host=None, logger=None, new=False):
+    def __init__(self,
+                 data: Union[str, dict]={},
+                 zfs: Optional[libzfs.ZFS]=None,
```

Comment:
i can't verify this on linux ☹

```
libiocage/lib/Jail.py:67: error: Name 'libzfs' is not defined
```

Reply:
which means we won't be able to verify this on Travis…

Reply:
Indeed that's a problem. Can we provide a Typing file for libzfs?